src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -102,14 +102,14 @@
   </xsl:param>
 
   <xsl:param name="title.font.family">
-    <xsl:call-template name="pickfont-serif"/>
+    <xsl:call-template name="pickfont-sans"/>
   </xsl:param>
 
   <!--
     Text properties
   -->
 
-  <xsl:param name="hyphenate">false</xsl:param>
+  <xsl:param name="hyphenate">true</xsl:param>
```

Comment:
I think we should keep this as an option that the user enables. It can be enabled by passing an XSL param to fopub. I guess that means that the fopub script needs to be updated to accept XSL params as input. You could also just consider adding a dedicated flag (i.e., `-H`) to enable this feature.